src/ydata_profiling/model/typeset_relations.py

```diff
@@ -24,7 +24,7 @@ def inner(series: pd.Series, *args, **kwargs) -> bool:
     return inner
 
 
-def string_is_bool(series: pd.Series, state: dict, k: Settings) -> bool:
+def string_is_bool(series: pd.Series, state: dict, k: Dict[str, bool]) -> bool:

```

Comment:
why the signature change?

Reply:
becouse as `k`, we are sending just `Setting.vars.bool.mappings`, not whole `Setting`